### ISSUE TITLE
[Request] RSS 

### ISSUE THREAD

**Issue by Jesse-Millwood** [fusion809/fusion809.github.io#1]
Really like your personal site,
Would be nice if you had an RSS feed. Granted, I don't know much about Jekyll there does seem to be some RSS plugins for Jekyll generated sites.

**Comment by fusion809** [comment]
Thanks for your interest. I have never used RSS before myself, but I just enabled the `jekyll-feed` plugin for Jekyll and my repo has a feed.xml file which according to this [extension's GitHub repo](https://github.com/jekyll/jekyll-feed) should contain the Atom feed.

**Comment by Jesse-Millwood** [comment]
Yeah I like rss because I can have an rss reader on my phone and or in
emacs and then I don't have to keep checking up on sites that interest me
because the rss feed lets me know when something new is there and you seem
to have to interesting and frequent content. Keep up the good work!
Jesse

**Comment by Jesse-Millwood** [comment]
Thanks for adding that so fast by the way

**Comment by fusion809** [comment]
Well I know the frustration of having to wait days for someone to respond to an issue at GitHub, so out of "do unto others" sort of attitude I made it go as fast as possible for ya. For my own ego-stroking, rofl, may I ask which post you liked the most?

**Comment by Jesse-Millwood** [comment]
Well I stumbled on your site because I was going to try and message you
about a stack overflow question that you asked about measuring the ram
usage for different DEs and then some how got on your site. I was switching
from XFCE and was looking for a newer comparison of different DEs that took
into account usage, not just features so that was refreshing to see because
I realized cinnamon was using a lot (10% CPU) for a DE on my limited
hardware (I ended up going with MATE). So that was my favorite, the
languages one was interesting too, it'll be cool to see them both expand.
Keep up the good work!
